views/components/ChartList.jsx

```diff
@@ -65,50 +65,64 @@ const ChartList = ({ user }) => {
           </TableRow>
         </TableHead>
         <TableBody>
-          {chartList.map((chart) => (
-            <TableRow key={chart._id}>
-              <TableCell align='center'>
-                <Link color='textPrimary' href={routes.chart(chart._id)}>
-                  {chart?.title?.toUpperCase()}
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                {chart?.description?.toUpperCase()}
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => onDuplicateChart(chart._id)}
-                >
-                  <PlaylistAdd />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Link href={routes.editChart(chart._id)} color='textPrimary'>
-                  <Edit />
-                </Link>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => handleShareChart(chart)}
-                >
-                  <Share />
-                </Button>
-              </TableCell>
-              <TableCell align='center'>
-                <Button
-                  type='button'
-                  variant='text'
-                  onClick={() => removeChart(chart._id)}
-                >
-                  <Delete />
-                </Button>
-              </TableCell>
-            </TableRow>
-          ))}
+          {chartList.map((chart) => {
+            const isDisabled = user.uid !== chart.owner
+            return (
+              <TableRow key={chart._id}>
+                <TableCell align='center'>
+                  <Link color='textPrimary' href={routes.chart(chart._id)}>
+                    {chart?.title?.toUpperCase()}
+                  </Link>
+                </TableCell>
+                <TableCell align='center'>
+                  {chart?.description?.toUpperCase()}
+                </TableCell>
+                <TableCell align='center'>
+                  <Button
+                    type='button'
+                    variant='text'
+                    onClick={() => onDuplicateChart(chart._id)}
+                  >
+                    <PlaylistAdd />
+                  </Button>
+                </TableCell>
+                <TableCell align='center'>
+                  <Link
```

Comment:
should we set the [disabled attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) on this link if it should be disabled?

Reply:
The material link component doesn't have the disabled property
https://v3.mui.com/api/link/
The button component does
https://v3.mui.com/api/button/
and the button component has an href property

What do you think should we use a button instead?

Reply:
The documentation states

> Any other properties supplied will be spread to the root element ([Typography](https://v3.mui.com/api/typography/)).

Since this uses an anchor tag by default we should be fine to pass the prop but you should verify it shows up in the UI too 😄